code/modules/organs/subtypes/augment.dm

```diff
@@ -0,0 +1,401 @@
+/obj/item/organ/internal/augment
```

Comment:
Few issues with these being internal organs:
- when they die, the scanner will report them as necrotic and decaying;
- they'll get infections i think;
- they'll get scarring when repaired;
- they'll get damaged by toxloss after kidneys and liver;
- fevers will damage them;
- they'll get magically repaired by psionics ("You encourage the damaged tissue of \the [I] to repair itself.")
- peri and shit will heal them;

Generally the internal organs weren't really made to be abused like this. Not too sure here.

Reply:
This sounds like an issue with robotic organs, to be fair.